spec/fixtures/acceptance/animal-example-simple/outcomes/outcome_safe_pet.txt

```diff
@@ -1,5 +1,9 @@
 # You picked a safe pet
 
+{{marker: is_a_safe_pet}}
```

Comment:
This isn't a pattern we'd use in the real world - two markers next to each other?

How about something like:
```
# You picked a safe pet
 
{{marker: is_a_safe_pet}}

$IF some_conditional

{marker: safe_pets_some_condition}}

Conditional content

$ENDIF
```

Then it demonstrates how the marker would normally be used, acting a bit like documentation. Plus the paired scenario could be updated to show the usage from scenario side, like:

```
 # And cats are amazing
 - question_1: cat
 outcome_safe_pet
has markers: is_a_safe_pet

 # And cats are amazing
 - question_1: cat
 outcome_safe_pet
has markers: is_a_safe_pet, safe_pets_are_cool
```